bgpd/bgp_mplsvpn.c

```diff
@@ -304,7 +304,7 @@ void vpn_leak_zebra_vrf_label_update(struct bgp *bgp, afi_t afi)
 
 	if (vpn_leak_to_vpn_active(bgp, afi, NULL, false)) {
 		ifp = if_get_vrf_loopback(bgp->vrf_id);
-		if (ifp && if_is_vrf(ifp) && if_is_up(ifp))
+		if (ifp && if_is_up(ifp))
```

Comment:
What if this is only a loopback, but not a VRF? `if_get_vrf_loopback()` can return true if this is only a loopback (default VRF).

Reply:
I observed a regression, the "import vpn"  command fails to install the bgpd label on the default loopback interface. `if_get_vrf_loopback()` returns VRF and default loopback, the check `if_is_vrf()` is unnecessary and incorrect, as it ignores the default loopback.